lib/internal/modules/esm/load.js

```diff
@@ -108,6 +132,51 @@ async function defaultLoad(url, context = kEmptyObject) {
     source,
   };
 }
+/**
+ * @typedef LoadContext
+ * @property {string} [format] Hint possibly returned by `resolve`
+ * @property {string | Buffer | ArrayBuffer} [source] source
+ * @property {Record<string, string>} [importAssertions] import attributes
+ */
+
+/**
+ * @typedef LoadReturn
+ * @property {string} format format
+ * @property {string} responseURL url
+ * @property {Buffer} source source
```

Comment:
I think `source` can be more than only a buffer, no?

Reply:
Yeah but `defaultLoad(Sync)` only returns `Buffer`s